package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.7.9-prerelease.1",
+  "version": "1.7.9-prerelease.2",
```

Comment:
actually not sure whether we need to increase the version here if only validation is changed

Reply:
I think we do since you'll need to re-pull this in the Coda repo.